packages/codemirror-graphql/package.json

```diff
@@ -13,34 +13,28 @@
     "url": "https://github.com/graphql/graphiql/issues?q=issue+label:codemirror-graphql"
   },
   "license": "MIT",
+  "main": "index.js",
+  "module": "esm/index.js",
   "files": [
-    "hint.esm.js",
-    "hint.js",
-    "hint.js.flow",
-    "info.esm.js",
-    "info.js",
-    "info.js.flow",
-    "jump.esm.js",
-    "jump.js",
-    "jump.js.flow",
-    "lint.esm.js",
-    "lint.js",
-    "lint.js.flow",
-    "mode.esm.js",
-    "mode.js",
-    "mode.js.flow",
+    "dist",
     "utils",
     "variables",
-    "results"
+    "results",
+    "/*.js",
```

Comment:
this strategy works but then we end up with `babel.config.js` and then `jest.config.js`. is it possible to mix include/ignore arguments now with files param?

Reply:
Excluded babel and jest configs